src/components/FlowRunDetails.vue

```diff
@@ -24,7 +24,7 @@
       </template>
     </p-key-value>
 
-    <p-key-value v-if="can.read.deployment && flowRun.deploymentId" label="Deployment" :alternate="alternate">
+    <p-key-value v-if="can.read.deployment && flowRun.deploymentId && deployment?.name" label="Deployment" :alternate="alternate">
```

Comment:
Can you move this to its own computed variable in the script? 
```ts
const showDeployment = computed(() => can.read.deployment && props.flowRun.deploymentId && deployment.value?.name)
```
ought to do it; it helps with template readability as these conditionals get longer and prevents syntax highlighting in web editors from breaking on the optional chain

Reply:
Also you probably don't actually care about checking that the name is defined (here or in the other file), you only care about the existence of the deployment at all

Reply:
I wasn't seeing the right behavior if I just did `&& deployment` -- what could I do differently to avoid the arbitrary choice of checking on `name`?

Reply:
Try:
```ts
import { isDefined } from '@/utilities'

const showDeployment = computed(() => can.read.deployment && isDefined(deployment.value))
```

Reply:
Done